connectomics/discfibers/ea_discfibers_merge_pathways.m

```diff
@@ -82,25 +87,46 @@
             
             if side == 1
                 side_name = 'right';
+                BIDS_side = '_model-ossdbs_hemi-R_tract-'; % this block is only executed for OSS-DBS
             else
                 side_name = 'left';
+                BIDS_side = '_model-ossdbs_hemi-L_tract-';
             end
-            % 
-            fiberActivation_file = ['fiberActivation_',side_name, '_', myFiles(k).name];
+
+
+            %BIDS notation
+            [~,subj_tag,~] = fileparts(obj.M.patient.list{sub});
+            subSimPrefix = [subj_tag, '_sim-'];
+            %fiberActivation_file = [subSimPrefix,'fiberActivation_',side_name, '_', myFiles(k).name];
+            fiberActivation_file = [subSimPrefix,'fiberActivation',BIDS_side, myFiles(k).name];
+            
+
+            %sub-sub-01_sim-fiberActivation_model-ossdbs_hemi-L_tract-stn2gpe_sm__mir_left.mat
```

Comment:
`sub-01` is a wrong subID here.
We also need to think about the naming of the tracts for multi-tract connectome, do we need to make sure that the name only contains `[a-zA-Z0-9]`

Reply:
I am not sure what is wrong here, we will need to discuss

Reply:
you mean the commented out part?

Reply:
yes, the commented out part. Just a reminder of the file naming pattern.

Reply:
There was some glitch in the original BIDS data I had worked on, so I noted this weird sub-sub-01, but later it did not occur